srcpkgs/sayonara-player/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'sayonara-player'
+pkgname=sayonara-player
+version=1.7.0.stable3
+revision=1
+build_style=cmake
+makedepends="pkg-config gst-plugins-bad1-devel qt5-devel qt5-tools-devel
+ qt5-plugin-mysql qt5-plugin-odbc qt5-plugin-pgsql qt5-plugin-sqlite
+ qt5-plugin-tds qt5-svg-devel taglib-devel"
+depends="qt5-core taglib gst-plugins-base1 gst-plugins-good1 gst-plugins-bad1
+ gst-plugins-ugly1 gst-libav"
```

Comment:
The `qt5-core`, `taglib`, `gst-plugins-base1` dependencies are automatically detected and shouldn't be in this list. Are `gst-plugins-bad1`, `gst-plugins-ugly1` and `gst-libav` strictly required for major functionality? [This FAQ](https://sayonara-player.com/faq/#i-cannot-play-ape-encoded-tracks-streams) suggests that `gst-libav` might enable optional support for particular encodings, and the gstreamer plugins probably don't matter from a program standpoint.

Reply:
thanks, will remove those 3 from depends

looking at the dev's provided pkgbuild file, they're probably optional dependencies